packages/cogify/src/cogify/cli/cli.cog.ts

```diff
@@ -63,6 +64,13 @@ export const BasemapsCogifyCreateCommand = command({
     ...logArguments,
     path: restPositionals({ type: Url, displayName: 'path', description: 'Path to covering configuration' }),
     force: flag({ long: 'force', description: 'Overwrite existing tiff files' }),
+    concurrency: option({
+      type: number,
+      long: 'concurrency',
+      description: 'Number of COGS to process at the same type',
+      defaultValue: () => 4,
```

Comment:
Would setting `defaultValue: () => os.cpus().length` be a worthwhile default? Or is that likely to do funky things in cloud nodes where "the number of CPU cores available" may be a more nebulous concept?

Reply:
The way we currently run this is we put 4+ containers on the same host, the host will have 100+ cores, I don't think we can run 400+ `gdal_translates` on the same host.

We generally only batch these containers into 10-20 tiffs per container so having a cpu().length would mean just run them all at the same time all the time!

Reply:
Ah yup, that makes sense.